es/reference/install.rst

```diff
@@ -70,26 +61,78 @@ Compilando la extensión:
 
 .. code-block:: bash
 
-    git clone git://github.com/phalcon/cphalcon.git
+    git clone --depth=1 git://github.com/phalcon/cphalcon.git
     cd cphalcon/build
     sudo ./install
 
-Añadiendo la extensión a php.ini
+Añadiendo la extensión a php.ini:
 
 .. code-block:: bash
 
+    # Suse: Add this line in your php.ini
+    extension=phalcon.so
+
+    # Centos/RedHat/Fedora: Add a file called phalcon.ini in /etc/php.d/ with this content:
+    extension=phalcon.so
+
+    # Ubuntu/Debian: Add a file called 30-phalcon.ini in /etc/php5/conf.d/ with this content:
+    extension=phalcon.so
```

Comment:
In my case:
```bash
/etc/php5/
├── apache2
│   └── conf.d
├── cli
│   └── conf.d
└── mods-available
```
Same problem on stackoverflow: http://stackoverflow.com/questions/24684144/php5-install-conf-d-folder-missing
So what should I do with it?
```rst
.. code-block:: bash

    # Suse: Add this line in your php.ini
    extension=phalcon.so

    # Centos/RedHat/Fedora: Add a file called phalcon.ini in /etc/php.d/ with this content:
    extension=phalcon.so

    # Ubuntu/Debian: Add a file called 30-phalcon.ini in /etc/php5/conf.d/ with this content:
    extension=phalcon.so

    # Debian with php5-fpm: Add a file called 30-phalcon.ini in /etc/php5/fpm/conf.d/ with this content:
    extension=phalcon.so
```